packages/zmq/zmq.5.0.0/opam

```diff
@@ -9,14 +9,12 @@ bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
 build: [
   ["jbuilder" "subst" "-p" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
-  ["jbuilder" "runtest" "-p" name "-j" jobs] {with-test}
```

Comment:
The failing tests has nothing to do with oUnit, but rather that libzmq has changed semantics (There is a PR against the zmq repo to fix the issue, and a new release will be made soon). 

Is the general policy that we disable tests when/if they fail? It seems a bit contradictory to me.

Reply:
I am not a packager on OPAM, so this is just IMHO (and I do prefer to have tests in general -- I am the upstream author of ounit). 

My take here is that a failing test on OPAM is non-actionnable. The best is to open an issue against ocaml-zmq repository, disable the test on OPAM and to re-enable the test on OPAM when the upstream release  will be made. An issue in ocaml-zmq repository is more actionnable for the ocaml-zmq maintainers than for the people uploading packages on OPAM and getting "red flags" because a random reverse-dependency is failing.

Does it make sense to you?

Reply:
I do understand that failing revdeps is annoying, but I think disabling tests here is a bit drastic.
It seems that there should be a general policy on what to do with failing revdeps, which seems slightly out of scope in this PR. 

In this particular case, its fine by me to disable tests on zmq 5.0, as a  new version will be arriving soon.